contracts/chain-adapters/WorldChain_Adapter.sol

```diff
@@ -42,9 +43,10 @@ contract WorldChain_Adapter is CrossDomainEnabled, AdapterInterface, CircleCCTPA
      */
     constructor(
         WETH9Interface _l1Weth,
+        IERC20 _l1Usdc,
         address _crossDomainMessenger,
         IL1StandardBridge _l1StandardBridge,
-        IERC20 _l1Usdc
```

Comment:
Why move this param around? Seems like it could cause errors

Reply:
I did it to group WETH and USDC together, and to then keep the OP contract addresses together. I've updated the deployment script so that it follows along, but can revert this if it's too risky.

Reply:
It's fine